R/aux_functions.R

```diff
@@ -45,6 +50,25 @@ inf_fn <- function(inc_samp = NULL, k = NULL) {
 #' @return a single `numeric` with the probability of extinction
 #' @export
 #' @inheritParams detect_extinct
+#'
+#' @examples
+#' res <- scenario_sim(
+#'   n.sim = 10,
+#'   num.initial.cases = 1,
+#'   prop.asym = 0,
+#'   prop.ascertain = 0.2,
+#'   cap_cases = 4500,
+#'   cap_max_days = 350,
+#'   r0isolated = 0.5,
+#'   r0community = 2.5,
+#'   disp.com = 0.16,
+#'   disp.iso = 1,
+#'   delay_shape = 1.65,
+#'   delay_scale = 4.28,
+#'   k = 0,
+#'   quarantine = FALSE
+#' )
+#' extinct_prob(res, cap_cases = 4500)
```

Comment:
ovf, yikes - bit noisy for a demo. does scenario sim have some convenient defaults, such that the example could be setting only a few arguments? alternatively, some data or inst object that could be loaded?

Reply:
also same note for `detect_extinct` - or better yet, merge their documentation via `@rdname`? they have the same arguments

Reply:
Currently `scenario_sim()` does not have many sensible defaults, it only has `r0subclin = NULL, disp.subclin = NULL`, so the noisy function parameterisation is required, for now.

Reply:
👍 - thoughts re single roxygen entry for both functions?

Reply:
I agree that it would be preferable, but I'm getting weird issue when using `@name` and `@rdname` where the function inheriting the documentation is not exported (not listed in `NAMESPACE`). 

As I'm not yet sure whether both `detect_extinct()` and `extinct_prob()` will both be exported after the refactor I propose we leave this for now and address in the future if both remain exported.